src/Sentry/Platforms/Cocoa/SentryOptions.cs

```diff
@@ -267,6 +267,72 @@ public void AddInAppInclude(string prefix)
             InAppIncludes ??= new List<string>();
             InAppIncludes.Add(prefix);
         }
+
+        /// <summary>
+        /// Options for experimental features in the native Sentry Cocoa SDK.
+        /// </summary>
+        public class NativeExperimentalOptions
```

Comment:
Once we've upgraded to 8.57.1 of the Cocoa SDK, we will need to add an `EnableSessionReplayInUnreliableEnvironment` option here as well, otherwise replay will automatically be disabled by the Cocoa SDK.

See: https://github.com/getsentry/sentry-cocoa/blob/main/CHANGELOG.md#8570